### PR TITLE
Use bbagent in devicelab tasks.

### PR DESCRIPTION
**Pull request by godofredoc**
This is part of a pending luci migration to the latest runner.

Bug: https://github.com/flutter/flutter/issues/79883

### CODE REVIEW

File: config/devicelab_config.star [hunk 1]
```diff
@@ -111,6 +111,7 @@ def devicelab_prod_config(branch, version, ref):
         name = drone_recipe_name,
         cipd_package = "flutter/recipe_bundles/flutter.googlesource.com/recipes",
         cipd_version = "refs/heads/master",
+        use_bbagent = True,
```
Comment (CaseyHillers) [inline config/devicelab_config.star:114]: Is this expected to be a no-op? Should we split prod and staging into separate PRs?

**Review by CaseyHillers — APPROVED** [review]
LGTM

File: config/devicelab_config.star (hunk 1 above)
Comment (godofredoc) [reply config/devicelab_config.star:114]: Adding the parameter to staging config causes recipe name conflicts .

Comment (CaseyHillers) [reply config/devicelab_config.star:114]: SG thanks for explaining
